samples/fortio_data/large.json

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
Is this file used anywhere?

Reply:
This is the data that generates the sample pngs in `samples/fortio_reports/`. I kept this data here so that any users of NH could compare the PNG with the underlying data. It also allows them to test running Fortio-UI on the data to generate interactive graphs, as documented in the README.md

Reply:
Ah, makes sense, sgtm!